app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -112,6 +113,12 @@ class Config {
 
   int get maxTaskRetries => 2;
 
+  /// The default number of commit shown in flutter build dashboard.
+  int get commitNumber => 25;
+
+  KeyHelper get keyHelper =>
```

Comment:
Since key helper is used throughout the code base, can you update all the constructions to just use this reference instead? This can be done in a separate PR.

Reply:
Yeah, I plan to do it in another PR.

Reply:
Can we add a TODO?

Reply:
Added.